style.css

```diff
@@ -783,7 +817,7 @@ textarea::placeholder {
 
   .sidep4 img {
     content: url("./Work/4.png");
-  }
+  } */

```

Comment:
- [ ] It is considered best practice to avoid keeping commented codes or unused codes in your work base, as it makes it messy. Kindly remove this unused part of your code. :wrench: